src/NzbDrone.Api/Indexers/ReleaseResource.cs

```diff
@@ -74,4 +75,97 @@ public DownloadProtocol DownloadProtocol
+            var parsedEpisodeInfo = model.RemoteEpisode.ParsedEpisodeInfo;
+            var remoteEpisode = model.RemoteEpisode;
+            var torrentInfo = (model.RemoteEpisode.Release as TorrentInfo) ?? new TorrentInfo();
+
+            // TODO: Clean this mess up. don't mix data from multiple classes, use sub-resources instead? (Got a huge Deja Vu, didn't we talk about this already once?)
+            return new ReleaseResource
+            {
+                Guid = releaseInfo.Guid,
+                Quality = parsedEpisodeInfo.Quality,
+                //QualityWeight
+                Age = releaseInfo.Age,
+                AgeHours = releaseInfo.AgeHours,
+                AgeMinutes = releaseInfo.AgeMinutes,
+                Size = releaseInfo.Size,
+                IndexerId = releaseInfo.IndexerId,
+                Indexer = releaseInfo.Indexer,
+                ReleaseGroup = parsedEpisodeInfo.ReleaseGroup,
+                ReleaseHash = parsedEpisodeInfo.ReleaseHash,
+                Title = releaseInfo.Title,
+                FullSeason = parsedEpisodeInfo.FullSeason,
+                SeasonNumber = parsedEpisodeInfo.SeasonNumber,
+                Language = parsedEpisodeInfo.Language,
+                AirDate = parsedEpisodeInfo.AirDate,
+                SeriesTitle = parsedEpisodeInfo.SeriesTitle,
+                EpisodeNumbers = parsedEpisodeInfo.EpisodeNumbers,
+                AbsoluteEpisodeNumbers = parsedEpisodeInfo.AbsoluteEpisodeNumbers,
+                Approved = model.Approved,
+                TemporarilyRejected = model.TemporarilyRejected,
+                Rejected = model.Rejected,
+                TvdbId = releaseInfo.TvdbId,
+                TvRageId = releaseInfo.TvRageId,
+                Rejections = model.Rejections.Select(r => r.Reason).ToList(),
+                PublishDate = releaseInfo.PublishDate,
+                CommentUrl = releaseInfo.CommentUrl,
+                DownloadUrl = releaseInfo.DownloadUrl,
+                InfoUrl = releaseInfo.InfoUrl,
+                DownloadAllowed = remoteEpisode.DownloadAllowed,
+                //ReleaseWeight
+
+                Seeders = torrentInfo.Seeders,
+                Leechers = (torrentInfo.Peers.HasValue && torrentInfo.Seeders.HasValue) ? (torrentInfo.Peers.Value - torrentInfo.Seeders.Value) : (int?)null,
+                Protocol = releaseInfo.DownloadProtocol,
+
+                IsDaily = parsedEpisodeInfo.IsDaily,
+                IsAbsoluteNumbering = parsedEpisodeInfo.IsAbsoluteNumbering,
+                IsPossibleSpecialEpisode = parsedEpisodeInfo.IsPossibleSpecialEpisode,
+                Special = parsedEpisodeInfo.Special,
+            };
+
+        }
+
+        public static ReleaseInfo ToModel(this ReleaseResource resource)
+        {
+            ReleaseInfo model;
+
+            if (resource.Protocol == DownloadProtocol.Torrent)
+            {
+                model = new TorrentInfo
+                {
+#warning FIXME: Missing on resource?
```

Comment:
Sounds like we don't use them, but I'm not opposed to adding them to have a more complete representation of the model in the resource.